app/services/find_empty_trainees.rb

```diff
@@ -28,9 +28,6 @@ class FindEmptyTrainees
     trainees.itt_end_date
     trainees.trn
     trainees.submitted_for_trn_at
-    trainees.withdraw_reason
```

Comment:
I think withdrawing only concerns non draft trainees but I can't remember why we (or possibly I) included the old fields in this list. I wonder if we need to account the the new ones via here https://github.com/DFE-Digital/register-trainee-teachers/blob/b74e8a97e14f6b3222d2cfe173a13e065553bfc6/app/services/find_empty_trainees.rb#L84

Reply:
@darokel  yep you can only withdraw non-draft trainees. I can update the method though like so to be sure?:

```ruby
def empty_draft_trainees
  # Finds all the draft trainees that do not have any degrees, disabilities and nationalities.
  trainees
    .draft
    .where(trainee_id: nil)
    .includes(:degrees, :disabilities, :nationalities, :withdrawal_reasons)
    .where(degrees: { id: nil }, disabilities: { id: nil }, nationalities: { id: nil }, withdrawal_reasons: { id: nil })
    .where(empty_fields_query)
end
```